kiwi/builder/container.py

```diff
@@ -57,6 +60,11 @@ def __init__(self, xml_state, target_dir, root_dir):
         self.target_dir = target_dir
         self.container_config = xml_state.get_container_config()
         self.requested_container_type = xml_state.get_build_type_name()
+        if not xml_state.build_type.get_derived_from():
+            self.base_image = None
+        else:
+            self.base_image = os.sep.join([root_dir, 'image', 'image_file'])
+
```

Comment:
I think we should change the way we retrieve the derived_from information. Actually the information in derived_from is an uri. Thus I suggest to write a method in xml_state.py which returns an uri object. e.g

```python
def get_derived_from_image_uri(self):
     ....code
```

* That method is then used in system/prepare.py and RootImport actually receives the Uri object
* That method is then called here and self.base_image is set via the Uri.translate()

I think that will make it easier for us to deal with different base image locations in the future. Additionally there will be only one place where we handle the result of derived_from

Thoughts ?